### PR TITLE
Feature/refactoring message parts

### PR DESCRIPTION
**Pull request by Opti213**
Refactoring
Update poetry.lock
Сollected attachments and entities in one module

### CODE REVIEW

File: botx/models/entities.py [hunk 1]
```diff
@@ -1,11 +1,34 @@
-"""Pydantic models for mentions."""
+"""Forward entities that can be received in message."""
```
Comment (lxmnk) [inline botx/models/entities.py:1]: `Entities that can be received in message`?

Comment (Opti213) [reply botx/models/entities.py:1]: that field has name `entities`

File: botx/models/entities.py [hunk 2]
```diff
@@ -17,6 +40,9 @@ class UserMention(BaseModel):
     #: name that will be used instead of default user name.
     name: Optional[str] = None
 
+    #: connection type via that entity was mention
```
Comment (lxmnk) [inline botx/models/entities.py:43]: Not a refactoring.

File: botx/models/entities.py [hunk 3]
```diff
@@ -55,7 +81,7 @@ def generate_mention_id(cls, mention_id: Optional[UUID]) -> UUID:  # noqa: N805
 
     @validator("mention_type", pre=True, always=True)
     def check_that_type_matches_data(
-        cls, mention_type: MentionTypes, values: dict,  # noqa: N805, WPS110
+            cls, mention_type: MentionTypes, values: dict,  # noqa: N805, WPS110
```
Comment (lxmnk) [inline botx/models/entities.py:84]: ??

File: pyproject.toml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 [tool.poetry]
 name = "botx"
-version = "0.15.17"
+version = "0.15.18"
+
```
Comment (lxmnk) [inline pyproject.toml:4]: Extra empty line.

File: tests/test_models/test_mentions.py [hunk 5]
```diff
@@ -14,11 +14,6 @@ def test_mention_id_will_be_generated_if_missed(mention_id):
     assert mention.mention_id is not None
 
 
-def test_error_when_no_mention_data():
```
Comment (lxmnk) [inline tests/test_models/test_mentions.py:17]: Please bring back test to its old location.

**Review by lxmnk — APPROVED** [review]

**Pull request merged**